src/Veldrid/Vk/VkTexture.cs

```diff
@@ -185,10 +185,16 @@ internal VkTexture(VkGraphicsDevice gd, ref TextureDescription description)
                     prefersDedicatedAllocation = false;
                 }
 
+                // Use "host cached" memory when available, for better performance of GPU -> CPU transfers
+                var propertyFlags = VkMemoryPropertyFlags.HostVisible | VkMemoryPropertyFlags.HostCoherent | VkMemoryPropertyFlags.HostCached;
+                if (TryFindMemoryType(_gd.PhysicalDeviceMemProperties, bufferMemReqs.memoryTypeBits, propertyFlags) == null)
```

Comment:
The comparison to `null` here does feel awkward in the presence of a "Try" function. When reading snippets like this, I expect the "Try" function to return a boolean.